libkbfs/md_id_journal.go

```diff
@@ -29,7 +29,8 @@ type mdIDJournal struct {
 // depend on the ID, as the mdJournal may change the ID when
 // converting to a branch.
 type mdIDJournalEntry struct {
-	ID MdID
+	ID            MdID
```

Comment:
Please update comment, too. I guess squash commits can still be converted to a non-squash branch, right?